base/src/org/compiere/model/MOrder.java

```diff
@@ -80,6 +80,9 @@
  * @author Yamel Senih, ysenih@erpya.com, ERPCyA http://www.erpya.com
  *		<a href="https://github.com/adempiere/adempiere/issues/1455">
  * 		@see FR [ 1455 ] Add Sales Region to Order and Invoice</a>
+ * @author Raul Capecce, raul.capecce@solopsoftware.com, Solop https://solopsoftware.com/
+ *		<a href="https://github.com/adempiere/adempiere/issues/4266">
```

Comment:
Hi @rcapecce, is there a reason to modify the comment in the `MOrder` class? I think the comment should be moved to the `MWFProcess` class.